### PR TITLE
feat: Add 'filter' query parameter to force using specific filter for conversion

### PR DESCRIPTION
**Pull request by dcamarasa**
Adds a 'filter' query parameter to be able to force using a specific filter for converting files.

### CODE REVIEW

File: src/utils/convertFile.ts [hunk 1]
```diff
@@ -9,6 +9,7 @@ import { unoserver } from './unoserver.js'
 export async function convertFile(
 	srcPath: string,
 	format: string,
+	filter?: string,
```
Comment (lynxtaa) [inline src/utils/convertFile.ts]: Could you please make the third argument as an options object? Like this:

```ts
srcPath: string,
format: string,
options?: {  filter?: string }
```

File: src/utils/unoserver.ts [hunk 2]
```diff
@@ -48,20 +48,26 @@ export class Unoserver {
 	 *
 	 * @param from source file
 	 * @param to target file
+	 * @param filter filter name to use during conversion process
 	 */
-	convert(from: string, to: string): Promise<void> {
+	convert(from: string, to: string, filter?: string): Promise<void> {
```
Comment (lynxtaa) [inline src/utils/unoserver.ts]: Could you please make the third argument as an options object? Like this:

```ts
from: string,
to: string,
options?: {  filter?: string }
```

File: src/utils/convertFile.ts (hunk 1 above)
Comment (dcamarasa) [reply src/utils/convertFile.ts]: Done

File: src/utils/unoserver.ts (hunk 2 above)
Comment (dcamarasa) [reply src/utils/unoserver.ts]: Done

**Comment by lynxtaa** [comment]
@dcamarasa I've published the updated version to DockerHub: `docker pull lynxtaa/unoserver-web:0.6.0`. Thanks for the PR!

**Comment by dcamarasa** [comment]
> @dcamarasa I've published the updated version to DockerHub: `docker pull lynxtaa/unoserver-web:0.6.0`. Thanks for the PR!

Thanks @lynxtaa ;)

**Pull request merged**